packages/ez_cmdliner/ez_cmdliner.0.1.1/opam

```diff
@@ -30,12 +27,18 @@ build: [
   ]
 ]
 depends: [
-  "ocaml" {>= "4.03.0"}
+  "ocaml" {>= "4.06.0"}
```

Comment:
Is this correct? Version 0.1.1 used to work fine with older compilers http://check.ocamllabs.io/

Reply:
done !